drivers/sensor/adi/adxl345/adxl345_stream.c

```diff
@@ -355,7 +355,18 @@ void adxl345_stream_irq_handler(const struct device *dev)
 	if (data->sqe == NULL) {
 		return;
 	}
-	data->timestamp = k_ticks_to_ns_floor64(k_uptime_ticks());
+
+	uint64_t cycles;
+
+	int rc = sensor_clock_get_cycles(&cycles);
```

Comment:
move `int rc` with the other variables at the top of the function, keep the assignment and check consecutive

```c
rc = sensor_clock_get_cycles(...);
if (rc !=0) {
...
```

mid-function declarations are normally discouraged, same in the rest of the code

Reply:
Done. thank!